unit_test/entities/test_esdl_object.py

```diff
@@ -81,13 +80,13 @@ def test_get_connected_assets(self):
         """Test for connection of two assets."""
         # Arrange
         asset = self.esdl_object.get_all_assets_of_type("producer")[0].esdl_asset
-        test_list1 = [("Pipe1_ret", Port.Out)]
-        test_list2 = [("Pipe1", Port.In)]
+        test_list1 = [("Pipe1_ret", "935fb733-9f76-4a8d-8899-1ad8689a4b12")]
```

Comment:
Wil je de port id hard coded hierin de test zetten?

Reply:
Het zou handig zijn om deze test uit te breiden met een systeem waarbij er meerdere assets op een port verbonden zijn.

Reply:
code id eruit gehaald

Reply:
Added test with multiple connection to one asset